### PR TITLE
[DC-1245] Point e2e tests to dev

### PR DESCRIPTION
**Pull request by s-rubenstein**
Moves the UI tests away from using the integration environment to use dev instead. Has the side effect of moving away from datarepo-actions and greatly simplifying this workflow.

### CODE REVIEW

File: .github/workflows/test-e2e.yml [hunk 1]
```diff
@@ -28,94 +30,22 @@ jobs:
...
-        env:
-          DEPLOYMENT_TYPE: 'ui'
-      - name: set cypresss env
-        run: |
-          eval $(cat env_vars)
-          echo "CYPRESS_BASE_URL=$(echo ${IT_JADE_API_URL})" >> $GITHUB_ENV
-          CYPRESS_GOOGLE_TOKEN=$(gcloud auth print-access-token)
+          base64 --decode <<< ${{ secrets.SA_B64_CREDENTIALS }} > ${GOOGLE_APPLICATION_CREDENTIALS}
+          gcloud auth activate-service-account --key-file ${GOOGLE_APPLICATION_CREDENTIALS}
+          gcloud auth list
```
Comment (snf2ye) [inline .github/workflows/test-e2e.yml]:
```suggestion
```
Leftover test code

**Review by snf2ye — APPROVED** [review]
Thanks - so exciting!

File: .github/workflows/test-e2e.yml [hunk 2]
```diff
@@ -28,94 +30,22 @@ jobs:
...
-      - name: initialize npm
+      - name: Cypress run
         env:
-          DEV_PROJECT: broad-jade-dev
+          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
           DISABLE_ESLINT_PLUGIN: true
         run: |
-          GCR_TAG=$(git rev-parse --short HEAD)
-          echo "GCR_TAG=$(echo ${GCR_TAG})" >> $GITHUB_ENV
+          PROXY_URL=https://jade.datarepo-dev.broadinstitute.org
```
Comment (fboulnois) [inline .github/workflows/test-e2e.yml]: I think it makes sense to pull this out into the `env` section.

**Review by fboulnois — APPROVED** [review]
One suggestion:

**Pull request merged**